services/viva-ms/src/lambdas/generateRecurringCaseHtml.ts

```diff
@@ -2,126 +2,189 @@ import to from 'await-to-js';
 import handlebars from '../helpers/htmlTemplate';
 import config from '../libs/config';
 import log from '../libs/logs';
-import { getItem as getStoredUserCase } from '../libs/queries';
+import { getItem } from '../libs/queries';
 import params from '../libs/params';
 import S3 from '../libs/S3';
+import {
+  GetObjectOutput as S3GetObjectOutput,
+  PutObjectOutput as S3PutObjectOutput,
+} from 'aws-sdk/clients/s3';
 
 import createRecurringCaseTemplate from '../helpers/createRecurringCaseTemplate';
 import putVivaMsEvent from '../helpers/putVivaMsEvent';
 import { getClosedUserCases, updateVivaCaseState } from '../helpers/dynamoDb';
+import { AWSError } from 'aws-sdk/lib/error';
+import { PromiseResult } from 'aws-sdk/lib/request';
 
 interface CaseItem {
+  id: string;
+  PK: string;
+  SK: string;
   updatedAt: number;
   forms: Record<
     string,
     {
       answers: unknown;
     }
   >;
+  currentFormId: string;
 }
 
-export async function main(event, context) {
-  const { caseKeys } = event.detail;
+interface FormAnswer {
+  field: {
+    id: string;
+    tags: string[];
+  };
+  value: unknown;
+}
+export interface LambdaRequest {
+  detail: {
+    caseKeys: {
+      PK: string;
+      SK: string;
+    };
+  };
+}
+
+export type LambdaResponse = boolean;
+
+export interface Dependencies {
```

Comment:
Is this the same as we're using in other lambdas - LambdaContext. See users-api/lambda/uploadAttachments for details